view/adminhtml/ui_component/async_events_logs_listing.xml

```diff
@@ -33,6 +33,7 @@
         <bookmark name="bookmarks"/>
         <columnsControls name="columns_controls"/>
         <filters name="listing_filters"/>
+        <filterSearch name="log_id" class="\Aligent\AsyncEvents\Model\Indexer\LuceneSearch"/>
```

Comment:
I noticed your previous commit had this as a `fulltext` search. Just curious if `log_id` is what you want to go back to here, based on the description you're able to search by a lot more than the `log_id`?

> Find all sales orders where customer email is gmail
> event_name: sales.order.* AND data.data.customer_email: "*@gmail.com"

> Find using specific order id in payload
> event_name: sales.order.created AND data.data.increment_id: *00017

 If you had problems when it was defined as `fulltext` I might be able to provide some help.